eng/common/scripts/Add-Issue-Comment.ps1

```diff
@@ -0,0 +1,53 @@
+[CmdletBinding(SupportsShouldProcess = $true)]
+param(
+  [Parameter(Mandatory = $true)]
+  [string]$RepoOwner,
+
+  [Parameter(Mandatory = $true)]
+  [string]$RepoName,
+
+  [Parameter(Mandatory = $true)]
+  [string]$IssueNumber,
+
+  [Parameter(Mandatory = $false)]
+  [string]$CommentPrefix,
+
+  [Parameter(Mandatory = $true)]
+  [string]$Comment,
+
+  [Parameter(Mandatory = $false)]
+  [string]$CommentPostFix,
+
+  [Parameter(Mandatory = $true)]
+  [string]$AuthToken
+)
+
+. "${PSScriptRoot}\logging.ps1"
+
+$headers = @{
+  Authorization = "bearer $AuthToken"
+}
+
+$apiUrl = "https://api.github.com/repos/$RepoOwner/$RepoName/issues/$IssueNumber/comments"
+
+$commentPrefixValue = [System.Environment]::GetEnvironmentVariable($CommentPrefix)
+$commentValue = [System.Environment]::GetEnvironmentVariable($Comment)
+$commentPostFixValue = [System.Environment]::GetEnvironmentVariable($CommentPostFix)
+
+if (!$commentPrefixValue) { $commentPrefixValue = $CommentPrefix }
+if (!$commentValue) { $commentValue = $Comment }
+if (!$commentPostFixValue) { $commentPostFixValue = $CommentPostFix }
+
+$PRComment = "$commentPrefixValue<br>$commentValue<br>$commentPostFixValue"
+
+$data = @{
+  body = $PRComment
+}
+
+try {
+  $resp = Invoke-RestMethod -Method POST -Headers $headers -Uri $apiUrl -Body ($data | ConvertTo-Json)
```

Comment:
I was playing with ```Invoke-RestMethod``` today on another script and noticed that you don't need to pack the headers yourself. Instead you can do this:

```powershell
Invoke-RestMethod [other args] -Authentication Bearer -Token $AuthToken
```

It'll automatically inject an auth header with a bearer token. Take a look at this PR for an example:

https://github.com/Azure/azure-sdk-tools/pull/1075/files#diff-5fddae178f9da7cfb4fcf5c5d337041bR34-R47